config/api.yml

```diff
@@ -1745,9 +1745,11 @@
     :identifier: physical_chassis
     :options:
     - :collection
+    - :subcollection
```

Comment:
I think this means that `physical_chassis` can be a subcollection of something else.

Reply:
yap, sorry I removed this